rsmtool/notebooks/header.ipynb

```diff
@@ -39,7 +39,6 @@
     "\n",
     "from functools import partial\n",
     "from os.path import abspath, exists, join\n",
-    "from textwrap import wrap\n",
```

Comment:
It got moved to `rsmtool.utils` since it's used in the new function I added: `get_subgroup_plot_params()`. I also tested reports after removing this to make sure nothing is broken.